### PR TITLE
Release 100.11.0

### PR DESCRIPTION
**Pull request by puneet-pdx**
Merges `develop` into `master` branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ You can use the Toolkit in your projects by:
```
Comment (gunt0001) [inline README.md:23]: In the "Features" section you should add a new entry for PopupView

**Review by gunt0001 — APPROVED** [review]
@puneet-pdx - looks good, one minor comment.

File: Documentation/Popup/README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
+# PopupView
+
+The PopupView shows a Popup's attribute list in a Recyclerview, allows users to edit a Popup's attribute list and delete the Popup. 
+
+
+## Workflow 
+
+The app can define the `PopupView` in its view hierarchy, as `PopupView` extends FrameLayout. The app 
+has to set the `popup` and `popupManager` property on the `PopupView` with the Popup it wasnts to view/edit. The `PopupView` toolkit component
```
Comment (eri9000) [inline Documentation/Popup/README.md:9]:
```suggestion
has to set the `popup` and `popupManager` property on the `PopupView` with the Popup it wants to view/edit. The `PopupView` toolkit component
```

**Review by eri9000 — APPROVED** [review]
@puneet-pdx Looks good, 
just one small comment.

**Pull request merged**